### PR TITLE
 * Fix dotnetVersion file

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Fixes the KoreBuild.ps1 to look for the version file in the correct place.

CC @victorhurdugaci

### CODE REVIEW

File: build/KoreBuild.ps1 [hunk 1]
```diff
@@ -11,7 +11,7 @@ $koreBuildFolder = $PSScriptRoot
 $koreBuildFolder = $koreBuildFolder.Replace($repoFolder, "").TrimStart("\")
 
 $dotnetChannel = "beta"
-$dotnetVersion = Get-Content .\cli.version.win
+$dotnetVersion = Get-Content $koreBuildFolder +"\"+cli.version.win
```
Comment (victorhurdugaci) [inline build/KoreBuild.ps1:14]: Spaces...

**Comment by victorhurdugaci** [comment]
:shipit:

**Comment by BrennanConroy** [comment]
Could you do this for the .sh too

**Comment by BrennanConroy** [comment]
@ryanbrandenburg we need this in the shell script

**Pull request merged**